platform/client/package.json

```diff
@@ -14,7 +14,7 @@
     "keycloak-js": "9.0.3",
     "offix-cache": "0.15.5",
     "offix-client": "0.15.5",
-    "offix-datastore": "0.2.0",
+    "offix-datastore": "0.3.4-dev2",
```

Comment:
Let's not use dev2 or anything like it since this is alpha and we have been making things working so we want to push them to latest anyway.

Reply:
@wtrocki okay, so should I be releasing the changes as an alpha? Like `0.3.4-alpha1`?

Reply:
0.3.4 etc. Merge this one. but in future just do regular bumps